r/configure

```diff
@@ -168,6 +162,17 @@ if [ "$ARROW_R_CXXFLAGS" ]; then
 fi
 
 # Test configuration
+CXXCPP="`${R_HOME}/bin/R CMD config CXX11` -E"
```

Comment:
Would it hurt to have this in the `if [TRUE]` block to match what's in the `else` block?

Reply:
If it succeeded, then it's already set. I'll add a comment.